### DISCUSSION TITLE
Can't update GameCube internal blueretro ota

### DISCUSSION THREAD

**Discussion by Gkjasl** [darthcloud/BlueRetro#569] [Q&A] [answered]
![Screenshot_20230120-113847](https://user-images.githubusercontent.com/123192734/213755121-4677658e-47c1-455c-8f84-f2b7de14db1d.png)
Hey I didn't find any discussion of this in my search, I'm having trouble with my GameCube blueretro ota via Android. My nearby PC doesn't even detect the blueretro, so that's out. I did manage to update to the December firmware over the air with Android, but now that it's updated to that firmware I can't do anything. The OTA page status stays stuck at this. I don't want to reset the firmware on the device in case it won't update after I reset to stock.

**Comment by darthcloud** [comment]
Try with another device like a laptop.

**Reply by Gkjasl** [reply]
> Requesting Bluetooth Device...
Connecting to GATT Server...
Getting BlueRetro Service...
> Bluetooth Device disconnected
NetworkError
Argh! Couldn't connect to BlueRetro


These are the results from a laptop running windows 10. I'll try resetting the firmware if there are no other solutions available.

**Reply by Gkjasl** [reply]
No luck, I reset the firmware to stock. I still can't connect to my blueretro via Android Bluetooth. Grimacing over this one, seems broken. If this isn't a known issue I'll contact laserbear industries, too.

**Reply by FerrazCezar** [reply]
I'm having a similar problem with the RetroScaler N64 Blueretro adapter. When I try to connect to the "BlueRetro Advance config" this message appears:

> Reading App version...
> Reading Api version...
> Requesting Bluetooth Device...
> Connecting to GATT Server...
> Getting BlueRetro Service...
> ABI version: 2
> Init Cfg DOM...
> Get Global Config CHRC...
> Reading Global Config...
> Argh! NotSupportedError: GATT operation failed for unknown reason.
> Bluetooth Device disconnected

Seems like a lot of adapters from different brands are having similar problems. This is very weird...

**Comment by Gkjasl** [comment, marked as answer]
So I think I found the solution. I turned my Android watch's Bluetooth off and I successfully paired with the blueretro. Completely forgot my watch has a Bluetooth signal. I'm using a Pixel watch if that helps anyone.

**Reply by darthcloud** [reply]
Thanks for taking time sharing your solution!
